### PR TITLE
Work on vendor listing

### PR DESCRIPTION
**Pull request by bmrsny**
Ready to go

### CODE REVIEW

File: app/views/partials/_supplier_list.html.erb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<div class="supplier_list">
+	<% @suppliers.each do |supplier| %>
+		<%= form_for(supplier) do |f| %>
+			<%= check_box(:current_suppliers, "suppliers", {:multiple => true}, supplier.name, nil)  %>
```
Comment (rshea303) [inline app/views/partials/_supplier_list.html.erb]: Are we still moving forward with the checkbox?

File: spec/models/listing_spec.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Listing, :type => :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```
Comment (rshea303) [inline spec/models/listing_spec.rb:4]: do we want a pending test merged to master?

Comment (bmrsny) [inline spec/models/listing_spec.rb:4]: yes

File: app/views/suppliers/show.html.erb [hunk 3]
```diff
@@ -1,3 +1,14 @@
 <div id="supplier_home">
-  <h1><%= @supplier.name %> Supplier Page</h1>
+	<h1><%= @supplier.name %> Supplier Page</h1>
+	<ul>
+		<% @listings.each do |listing|  %>	
+			<li>
+				<% item = Item.find(listing.item_id)  %>
```
Comment (rshea303) [inline app/views/suppliers/show.html.erb:6]: Do we want to push this logic into the model?

File: spec/models/item_spec.rb [hunk 4]
```diff
@@ -74,4 +74,13 @@
     item.price = -3
     expect(item).to_not be_valid
   end
+
+	xit "should have a supplier" do
```
Comment (rshea303) [inline spec/models/item_spec.rb:78]: Are we okay merging a 'skipped' test?

File: app/views/suppliers/show.html.erb (hunk 3 above)
Comment (bmrsny) [inline app/views/suppliers/show.html.erb]: Did we not work on this together

Comment (rshea303) [inline app/views/suppliers/show.html.erb:6]: Yes, we did.  But it seemed like a very sophisticated comment to make.

Comment (bmrsny) [inline app/views/suppliers/show.html.erb:6]: I feel like this logic should be linked to the item. So Suppliers can call on items.
supplier.items that would return all items for a supplier
but we spoke with another team member who didn't not believe this was the way to handle the relationship. So I would like to see which fiinal implementation makes the team happy before refactoring

Comment (rshea303) [inline app/views/suppliers/show.html.erb:6]: I think we should merge it since all tests are passing and then we can focus on this during refactoring.

**Pull request merged**